pandokia/db_psycopg2.py

```diff
@@ -168,7 +168,7 @@ def next(self, sequence_name):
     su postgres
 
     psql postgres
-        \password postgres
+        \\password postgres
```

Comment:
This is in a comment. Is this needed?

Reply:
For some reasons, python think that it was regular expression even though it is in block comment.